hugr_validator/Cargo.toml

```diff
@@ -1,11 +1,9 @@
 [package]
 name = "hugr_validator"
-version = "0.1.0"
+version = "0.3.0"
```

Comment:
Why skip v0.2.0?

Reply:
Good point! Behind the scenes, I used 0.2 for the version that used hugr v0.4.0